lua/nvim-treesitter/health.lua

```diff
@@ -51,6 +52,7 @@ function M.checkhealth()
       health_ok(parser_name .. " parser found.")
 
       locals.checkhealth(parser_name)
+      highlight.checkhealth(parser_name)
```

Comment:
I'm definitely not for adding checkhealth functions inside modules. Every check should be done inside the health file in my opinion. It's easier when debugging,and you might not need to import anything.

Reply:
also locals.checkhealth and highlight.checkhealths uses the same get_query fonction, so less imports makes the code easier to read.

Reply:
Hmm okay, i'm going to refactor everything inside the health function then.